demo/admin/src/products/future/generated/ProductForm.tsx

```diff
@@ -131,6 +136,7 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
         const output = {
             ...formValues,
             category: formValues.category?.id,
+            priceRange: formValues.priceRange ? { min: parseFloat(formValues.priceRange.min), max: parseFloat(formValues.priceRange.max) } : null,
```

Comment:
why this conversion (also for mutation)?

you also didn't need it in #2593 (which should be the base for this)

Reply:
You're right, I don' t need it. I will remove it.